### ISSUE TITLE
some analyzer errors are not reported

### ISSUE THREAD

**Issue by sigmundch** [dart-lang/dev_compiler#25]
For example, this error is not reported by ddc, but reported when invoking the analyzer from the command line:
```dart
abstract class B {
  m();
}
class A implements B {}
```

The analyzer indicates that `A` should be abstract or implement `m`.

**Comment by jmesserly** [comment]
fwiw, I checked yesterday and this still reproduces. Very strange.

**Comment by jmesserly** [comment]
aha. We're not getting errors from ErrorVerifier:
```dart
/**
 * A visitor used to traverse an AST structure looking for additional errors and
 * warnings not covered by the parser and resolver.
 */
class ErrorVerifier extends RecursiveAstVisitor<Object> {
```

fwiw, that description sounds a lot like another visitor we have: CodeChecker!

**Comment by jmesserly** [comment]
yeah, it turns out very many errors/warnings/hints were not reported. wow. https://codereview.chromium.org/1183733004/
